legacy/README.md

```diff
@@ -17,7 +17,7 @@ You can run jobs using the bootstrap.py script like so:
 ```
 $ bootstrap/bootstrap.py \
     --job=pull-cert-manager-verify \
-    --repo=github.com/jetstack/cert-manager=master \
```

Comment:
👍 although a follow up may be to remove this legacy stuff :) 🎉  pretty sure bootstrap.py is no longer used anywhere